File: app/src/main/java/com/example/myfirstapp/model/UserInfo.java
package com.example.myfirstapp.model;

public class UserInfo {
    private String name;
    private String email;
    private String password;
    private String confirm_password;
    public String getEmail()
    {
        return email;
    }

    public void setEmail(String email)
    {
        this.email = email;
    }

    public String getPassword()
    {
        return password;
    }

    public void setPassword(String password)
    {
        this.password = password;
    }
    public String getName()
    {
        return name;
    }
    public void setName(String name)
    {
        this.name=name;
    }
    public String getConfirm_password()
    {
        return confirm_password;
    }
    public void setConfirm_password(String confirm_password)
    {
        this.confirm_password=confirm_password;
    }
}

File: app/src/main/java/com/example/my_first_application/model/UserInfo.kt
package com.example.my_first_application.model

class UserInfo {
    private var name: String? = null
    private var email: String? = null
    private var password: String? = null
    private var confirmPassword: String? = null
    fun setName(name: String?) {
        this.name = name
    }
    fun setEmail(email: String?) {
        this.email = email
    }
    fun setpassword(password: String?) {
        this.password = password
    }
    fun setConfirmPassword(confirm_password: String?) {
        this.confirmPassword = confirm_password
    }
    fun getName():String?{
        return name
    }
    fun getEmail(): String? {
        return email
    }
    fun getPassword():String?{
        return password
    }
    fun getConfirmPassword():String?{
        return confirmPassword
    }
}